specs/vep-2448-vdk-run-logs-simplified-and-readable/README.md

```diff
@@ -381,6 +661,36 @@ integratable with Notebook
 
 ### [Progress Indicators](https://github.com/vmware/versatile-data-kit/milestone/17)
 
+
+#### Implement ProgressTracker in vdk-core
+Description: As a user, I want the tracking system to automatically initialize when I start a CLI execution so that I don't have to manually set it up.
+Acceptance Criteria: Progress tracking starts automatically with CLI execution.
+#### Create Child Tracker for New Steps
+escription: As a user, I want to see the progress for each specific step in my task so that I can understand how much work remains.
```

Comment:
```suggestion
Description: As a user, I want to see the progress for each specific step in my task so that I can understand how much work remains.
```